docs/source/optim.rst

```diff
@@ -146,6 +146,38 @@ allows dynamic learning rate reducing based on some validation measurements.
 Learning rate scheduling should be applied after optimizer's update; e.g., you
 should write your code this way:
 
+Example::
+
+    scheduler = ExponentialLR(optimizer, gamma=0.9)
+    for epoch in range(20):
+        for input, target in dataset:
+            optimizer.zero_grad()
+            output = model(input)
+            loss = loss_fn(output, target)
+            loss.backward()
+            optimizer.step()
+            scheduler.step()
+
+One interesting feature of schedulers are that they can be called back-to-back and to
+get chained. This will lead to by schedulers are getting applied simultaneously.
+
+Example::
+
+    scheduler1 = ExponentialLR(optimizer, gamma=0.9)
+    scheduler2 = MultiStepLR(optimizer, milestones=[30,80], gamma=0.1)
+    for epoch in range(20):
+        for input, target in dataset:
+            optimizer.zero_grad()
+            output = model(input)
+            loss = loss_fn(output, target)
+            loss.backward()
+            optimizer.step()
+            scheduler1.step()
+            scheduler2.step()
+
+In many places during this tutorial, we will use the following template to refer to schedulers
```

Comment:
nit: this will be part of the documentation, and not a tutorial :)
```suggestion
In many places in the documentation, we will use the following template to refer to schedulers
```